assets/sass/v2/_forms.scss

```diff
@@ -8,4 +8,11 @@
     margin: 25px 0 10px;
     text-align: left;
   }
+
+  .siw-main-body {
+    .ion-messages-container {
```

Comment:
this will break the message when it renders in callout .

Reply:
why is that ?

Reply:
callout is rendered from the view and it should not even get this `ion-messages-container` className iirc

Reply:
the callout render at IdentifyView for unknown user messages.